myko/src/lib/visualisation/sketch.js

```diff
@@ -154,7 +169,20 @@ function checkForAdds(p5, addedActivs) {
 
   if (!addedActivs) {
     console.log('no activities yet');
+    weatherOn = true;
+    weatherPosition = 1;
```

Comment:
Here I tried to make it easy to decide where the weather should be placed and what kind, snow for nasims katter and rain for tillverka aktivitet for now. I guess one could skip the cloud and just let it rain over half the screen for rain and the other half snow, could be an easy good enough fix for now?

Reply:
Yeah, that could work as well! But again, you probably could try to use the activity image position arrays if you'd want to use the cloud for snow and tea things for rain, for example